WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -37,7 +40,8 @@ import org.wordpress.android.widgets.PostListButtonType.BUTTON_SYNC
 import org.wordpress.android.widgets.PostListButtonType.BUTTON_VIEW
 import javax.inject.Inject
 
-private const val MAX_NUMBER_OF_VISIBLE_ACTIONS = 3
+private const val MAX_NUMBER_OF_VISIBLE_ACTIONS_STANDARD = 3
+private const val MAX_NUMBER_OF_VISIBLE_ACTIONS_COMPACT = Int.MAX_VALUE
```

Comment:
This seems a bit confusing, especially since the number of visible actions on the compact view is `1`.
I don't want to re-open [this discussion](https://github.com/wordpress-mobile/WordPress-Android/pull/9657#discussion_r277591246) but I'd like to hear your opinion. What I had in mind is basically create the following structure

Standard (SingleItem,SingleItem,MoreItem(SingleItem,SingleItem,...))
Compact(MoreItem(SingleItem,SingleItem,...))

What do you think about it?

Reply:
Ah I see what you mean. Involved some slight re-jigging, but that is cleaner. And you're right - the menu is effectively a more menu in compact - it's all hidden behind an ellipsis which is the definition of a more menu in the standard view.